spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly fix the failed tests.
![image](https://user-images.githubusercontent.com/24703171/193554339-a5328099-e12a-4fc3-b3f7-b971771c10d9.png)

Reply:
Thank you for the update, I have fixed it now.